yt/data_objects/particle_trajectories.py

```diff
@@ -142,9 +148,11 @@ def has_key(self, key):
     def keys(self):
         return self.field_data.keys()
 
-    def _get_full_field_name(self, field):
-        ds_first = self.data_series[0]
-        dd_first = ds_first.all_data()
+    def _get_full_field_name(self, field, ds_first=None, dd_first=None):
```

Comment:
Since this function is called exactly ONCE in the whole code base _and_ is private, I don't think it's worth giving default values to arguments here (even if they make perfect sense)
What's more is that we only ever use it to retrieve the `ftype` part of the full name, so might as well align the function's return value (and name) with its actual usage.